### PR TITLE
Move reader study scoring to a celery task

### PR DESCRIPTION
**Pull request by MikeOverkamp-diag**

### CODE REVIEW

**Comment by jmsmkn** [comment]
Please rebase this PR on to master to get the CI changes

File: app/grandchallenge/reader_studies/signals.py [hunk 1]
```diff
@@ -44,23 +44,10 @@ def update_image_permissions(instance, action, reverse, model, pk_set, **_):
...
-        ground_truth = Answer.objects.filter(
-            question=instance.question,
-            is_ground_truth=True,
-            images__in=pk_set,
-        ).first()
-        if ground_truth:
-            instance.calculate_score(ground_truth.answer)
-            instance.save()
+    add_scores.apply_async(
+        kwargs={"instance_pk": instance.pk, "pk_set": list(pk_set)}
```
Comment (jmsmkn) [inline app/grandchallenge/reader_studies/signals.py]: Are you certain that this works? Even outside of automated tests? I thought that sometimes you get problems with Celery and serialisation of the kwargs, so you might need to use `str(instance.pk)` and ensure that `list(pk_set)` is a list of strings.

File: app/grandchallenge/reader_studies/signals.py [hunk 2]
```diff
@@ -44,23 +44,10 @@ def update_image_permissions(instance, action, reverse, model, pk_set, **_):
 
 @receiver(m2m_changed, sender=Answer.images.through)
 def assign_score(instance, action, reverse, model, pk_set, **_):
+    from grandchallenge.reader_studies.tasks import add_scores
```
Comment (jmsmkn) [inline app/grandchallenge/reader_studies/signals.py]: Does this need to be a local import?

File: app/grandchallenge/reader_studies/signals.py (hunk 1 above)
Comment (MikeOverkamp-diag) [reply app/grandchallenge/reader_studies/signals.py]: I did not run into any issues when testing it by hand, but I added a cast to string just to be sure.

File: app/grandchallenge/reader_studies/signals.py (hunk 2 above)
Comment (MikeOverkamp-diag) [reply app/grandchallenge/reader_studies/signals.py]: No, changed.

**Pull request merged**